projects/vdk-control-cli/src/vdk/internal/control/command_groups/job/download_job.py

```diff
@@ -38,6 +38,16 @@ def download(self, team: str, name: str, path: str):
                 )
 
             log.info(f"Downloaded Data Job in {path}/{name}")
+
+            log.info(f"Downloading keytab...")
+            keytab_file_path = os.path.join(path, f"{name}.keytab")
+            response = self.api.get_jobs_api().data_job_keytab_download_with_http_info(
+                team_name=team, job_name=name, _preload_content=False
+            )
+            with open(keytab_file_path, "wb") as w:
+                w.write(response.raw_data)
+            log.info(f"Saved keytab in {keytab_file_path}")
```

Comment:
encapsulate this into a separate method.

Reply:
Add appropriate error handling. 

If the download-key fails we do not want the whole Download job to be considered failure. 
Keys are to mandatory and jobs may work without them. Only for some functionalities and in some cases are they used.

It would be more appropriate to post a warning to user that they do not have key.